src/applications/mhv-medical-records/tests/reducers/refresh.unit.spec.js

```diff
@@ -27,18 +27,19 @@ describe('safeNewDate', () => {
 });
 
 describe('getPhase', () => {
-  it('should return null when extractStatus is invalid', () => {
-    const result = getPhase(null, Date.now());
+  it('should return null if any required field in extractStatus is missing', () => {
+    const extractStatus = {}; // Missing all required fields
+    const result = getPhase(extractStatus, Date.now());
     expect(result).to.be.null;
   });
 
-  it('should return null when retrieved timestamp is invalid', () => {
+  it('should return null if extractStatus is missing required fields', () => {
     const extractStatus = {
       lastRequested: Date.now(),
-      lastCompleted: Date.now(),
+      lastCompleted: null,
       lastSuccessfulCompleted: Date.now(),
     };
-    const result = getPhase(extractStatus, null);
+    const result = getPhase(extractStatus, Date.now());
     expect(result).to.be.null;
   });
```

Comment:
Why did you replace these unit tests? You shouldn't replace old tests with new ones--you should just add the new tests. Also I think you rolled back your changes to the `getPhase` function, so you shouldn't have any new tests for it.